src/masternodes/mn_checks.cpp

```diff
@@ -3348,6 +3348,70 @@ bool IsMempooledCustomTxCreate(const CTxMemPool & pool, const uint256 & txid)
 
 std::vector<DCT_ID> CPoolSwap::CalculateSwaps(CCustomCSView& view) {
```

Comment:
Please combine CalculateSwaps and TextExecuteSwap and toggle with a bool.

Reply:
Okay